octopoes/server/server.py

```diff
@@ -59,6 +103,36 @@ def health(self) -> Any:
 
         return response
 
+    def get_ingesters(self) -> Any:
+        """List ingesters."""
+        return [IngesterModel(id=ingester) for ingester in self.ingesters.keys()]
+
+    def get_graphql_schema(self, ingester_id: str) -> Any:
+        """Serve graphql schema."""
+        if ingester_id not in self.ingesters:
+            return status.HTTP_404_NOT_FOUND
+
+        return print_schema(self.ingesters[ingester_id].current_schema.full_schema)
+
+    def get_graphiql(self, ingester_id: str) -> Any:
+        """Serve graphiql frontend."""
+        if ingester_id not in self.ingesters:
+            return status.HTTP_404_NOT_FOUND
+
+        # load template from disk
+        graphiql_template = Path(__file__).parent / "static" / "graphiql.html"
+        with open(graphiql_template, "r") as graphiql_html:
```

Comment:
See also python string templates, could be useful in your case?

Reply:
Fixed!